_pytest/doctest.py

```diff
@@ -43,6 +43,10 @@ def pytest_addoption(parser):
         action="store_true", default=False,
         help="ignore doctest ImportErrors",
         dest="doctest_ignore_import_errors")
+    group.addoption("--doctest-encoding",
```

Comment:
I think this would be better off being a `ini` option instead:

1. If one has to use this option, then usually all `doctest.txt` files will have the same encoding (if the developers want to keep their sanity that is) so it makes sense to configure that in `pytest.ini`;
2. If really desired, one can use the `-o` option to override `ini` options in the command-line.
3. It is usually not a good idea to require users to pass certain command-line options to make the test suite pass;